content/01.abstract.md

```diff
@@ -1,11 +1,16 @@
 ## Abstract
 
-Professional societies and the conferences that they manage provide an important venue for the dissemination of scientific knowledge.
-Being invited to deliver a keynote at an international society meeting or named a fellow of such a society is a major recognition.
-We sought to understand the extent to which such recognitions reflected the composition of their corresponding field.
-We collected keynote speaker invitations for the international meetings held by the International Society for Computational Biology as well as the names of Fellows, an honorary group within the society.
-We compared these honorees with last and corresponding author contributions in field-specific journals.
-We used multiple methods to estimate the race, ethnicity, gender, and name groupings of authors and the recipients of these honors.
-To address weaknesses in existing approaches, we built a new dataset of more than 700,000 people with name-nationality pairs from Wikipedia and trained long short-term memory neural networks to make predictions.
-Although we observe similar gender distributions between the group of honorees and authors, the proportion of female scientists in the field has not yet reached parity.
-Through each method, we find that white scientists are overrepresented among speakers and honorees, while scientists of color are underrepresented.
+Professional societies and their conferences provide an important venue for disseminating scientific knowledge.
+Delivering a keynote for or being named a fellow by an international society is a major recognition.
+We assess whether such recognitions reflect the composition of the field of bioinformatics.
+We compiled a list of 412 _International Society for Computational Biology_ honorees (75 fellows and 337 keynote speakers).
```

Comment:
```suggestion
We compiled a list of 412 _International Society for Computational Biology_ honorees (75 fellows and 337 keynote speakers) along with their affiliations at the time they received the honor.
```

Reply:
This is an important detail, but I think due to space limits is best described in the main text.